<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerTCPClass</class>
 <widget class="QMainWindow" name="ServerTCPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerTCP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#lblConnectionStatus[state = &quot;0&quot;]{
	background-color: rgb(255, 0, 0);

}
#lblConnectionStatus[state = &quot;1&quot;]{
	background-color: rgb(85, 255, 127);

}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>258</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="lstConsole"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>271</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Server Information</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spnServerPort">
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnStartServer">
          <property name="text">
           <string>Start Server</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblConnectionStatus">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="state" stdset="0">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Message Information</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Message</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lnMessage"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="btnSendToAll">
          <property name="text">
           <string>Send To All</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ServerTCP.qrc"/>
 </resources>
 <connections/>
</ui>
